### DISCUSSION TITLE
Checkbox keyboard accessibility

### DISCUSSION THREAD

**Discussion by keithburgie-rts** [chakra-ui/chakra-ui#3491] [Q&A]
I'm able to tab through checkboxes, but not check or uncheck them via spacebar. As mentioned in the docs, "Native HTML checkboxes are 100% accessible by default..." Any way I can restore this functionality? It does not work for examples on the docs page: https://chakra-ui.com/docs/form/checkbox

**Comment by dodas** [comment]
I just tried on Mac OS in Brave (Chromium based), Firefox as well as Safari and checkbox toggling with spacebar works in all of them.

**Reply by keithburgie-rts** [reply]
Any idea what might be going on? It doesn't work for me in any browser on the docs page or in my application. But when I go anywhere else, like here, https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_checkbox, spacebar works to check and uncheck boxes.

**Reply by with-heart** [reply]
What happens if you try it in incognito mode?

**Comment by benmcginnis** [comment]
It looks like the example custom checkbox does not work on the chakra docs for the `useCheckbox` hook: https://chakra-ui.com/docs/hooks/use-checkbox as you can see in this loom video: https://www.loom.com/share/6883977835f545ab9b9209168363f86b?sid=a035d499-8fd5-4967-be24-d9e98caa321e

**Reply by veloware** [reply]
remove the `hidden` attribute from the `input` element
